src/tests/ftest/pool/evict_test.py

```diff
@@ -178,14 +181,14 @@ def test_evict(self):
         """
         pool = []
         container = []
-        non_pool_servers = []
+        #non_pool_servers = []
         # Target list is configured so that the pools are across all servers
         # except the pool under test is created on half of the servers
         pool_tgt = [num for num in range(len(self.hostlist_servers))]
         pool_tgt_ut = [num for num in range(int(len(self.hostlist_servers)/2))]
         tlist = [pool_tgt, pool_tgt, pool_tgt_ut]
         pool_servers = [self.hostlist_servers[:len(tgt)] for tgt in tlist]
-        non_pool_servers = [self.hostlist_servers[len(tgt):] for tgt in tlist]
+        #non_pool_servers = [self.hostlist_servers[len(tgt):] for tgt in tlist]
```

Comment:
Same here.

Reply:
Same as above. If we don't comment, it would cause a pylint error.